src/status_im/contexts/communities/events.cljs

```diff
@@ -59,6 +59,9 @@
       (update :chats <-chats-rpc)
       (update :token-permissions seq)
       (update :categories <-categories-rpc)
+      (assoc :membership-permissions?
+             (some #(= (:type %) constants/community-token-permission-become-member)
```

Comment:
I think this logic should be unified, but it's being duplicated. See https://github.com/status-im/status-mobile/blob/ed3cb4257234f0917531d64c6543587ef42ed5fb/src/status_im/subs/communities.cljs#L311-L313

I would define a function in `data_store/communities.cljs`, unit test it and reuse it in the subscription I shared above, and also here.